src/AppInstallerCLICore/Commands/FontCommand.cpp

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "pch.h"
+#include "FontCommand.h"
+#include "Workflows/CompletionFlow.h"
+#include "Workflows/WorkflowBase.h"
+#include "Workflows/FontFlow.h"
+#include "Resources.h"
+
+namespace AppInstaller::CLI
+{
+    using namespace AppInstaller::CLI::Execution;
+    using namespace AppInstaller::CLI::Workflow;
+    using namespace AppInstaller::Utility::literals;
+    using namespace std::string_view_literals;
+
+    Utility::LocIndView s_FontCommand_HelpLink = "https://aka.ms/winget-command-help"_liv;
+
+    std::vector<std::unique_ptr<Command>> FontCommand::GetCommands() const
+    {
+        return InitializeFromMoveOnly<std::vector<std::unique_ptr<Command>>>({
+            std::make_unique<FontListCommand>(FullName()),
+        });
+    }
+
+    Resource::LocString FontCommand::ShortDescription() const
+    {
+        return { Resource::String::FontCommandShortDescription };
+    }
+
+    Resource::LocString FontCommand::LongDescription() const
+    {
+        return { Resource::String::FontCommandLongDescription };
+    }
+
+    Utility::LocIndView FontCommand::HelpLink() const
+    {
+        return s_FontCommand_HelpLink;
+    }
+
+    void FontCommand::ExecuteInternal(Execution::Context& context) const
+    {
+        OutputHelp(context.Reporter);
+    }
+
+    std::vector<Argument> FontListCommand::GetArguments() const
+    {
+        return {
+            Argument::ForType(Args::Type::FamilyName),
+            Argument::ForType(Args::Type::Moniker),
+            Argument::ForType(Args::Type::Source),
+            Argument::ForType(Args::Type::Tag),
+            Argument::ForType(Args::Type::Exact),
+            Argument::ForType(Args::Type::AuthenticationMode),
+            Argument::ForType(Args::Type::AuthenticationAccount),
+            Argument::ForType(Args::Type::AcceptSourceAgreements),
```

Comment:
* `-n / --count ` ?
* `--scope` ?
* `-q / --query` ? 

Will `FamilyName` be the default positional, so if a user typed `winget font list Arial` it would be the same as `winget font list --family-name Arial` ?

Reply:
Made `--family` positional. I left out the args you mentioned above until we fully support it just so that it's clear what a user can do at this moment.